onnxruntime/test/contrib_ops/col2im_test.cc

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <stdexcept>
+#include "gtest/gtest.h"
+#include "test/providers/provider_test_utils.h"
+#include "core/util/math.h"
+
+namespace onnxruntime {
+namespace test {
+
```

Comment:
Do we need an unnamed namespace to house the tests?

Reply:
No, why? (though some test do add an additional `contrib` namespace between `onnxruntime` and `test` ones)

You can refer to https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/test/contrib_ops to see what other tests do

Reply:
I see. Just checking since I wasn't sure